README.md

```diff
@@ -1,2 +1,68 @@
-# spark-client
-A lakeFS client for Apache Spark
+# lakeFS Spark Client
+
+Utilize the power of Spark to interact with the metadata on lakeFS. Possible use-cases include:
+
+* Create a DataFrame for listing the objects in a specific commit or branch.
+* Compute changes between two commits.
+* Export your data for consumption outside lakeFS.
+* [Something with Retention]
```

Comment:
Let's remove this, we managed to re-open requirements along the way.  Instead perhaps
```suggestion
* Bulk operations on underlying storage
```